test/goalc/source_templates/with_game/test-vf-constant-loading.gc

```diff
@@ -0,0 +1,14 @@
+(defun test-vector-constant ()
+  (let ((vector-1 (new 'stack 'vector)))
```

Comment:
Is this file used anywhere?

Reply:
my bad, was a left over when i was testing the new ASM instructions that have been decided against.